packages/react/src/components/Callout/CalloutContent.base.tsx

```diff
@@ -111,6 +111,37 @@ function useBounds(
   return getBounds;
 }
 
+/**
+ * (Hook) to return the maximum available height for the Callout to render into.
+ */
+function useMaxHeight(
+  { calloutMaxHeight, directionalHint, directionalHintFixed, hidden }: ICalloutProps,
+  positions?: ICalloutPositionedInfo,
+  getBounds?: () => IRectangle,
```

Comment:
Oops, sorry, I messed up here. After looking at it again it seems like `getBounds` should was returning `IRectangle` or `undefined` not being a function or `undefined`. If that is the case this should probably be:

```suggestion
  getBounds: () => IRectangle | undefined,
  positions?: ICalloutPositionedInfo,
```

Sorry about the confusion!

Reply:
oops, I misread that too 😅